alphastats/statistics/DifferentialExpressionAnalysis.py

```diff
@@ -179,15 +187,20 @@ def _ttest(self) -> pd.DataFrame:
         # calculate fold change (if its is not logarithimic normalized)
         mat_transpose = self.mat.transpose()
 
+        if not self.preprocessing_info[PreprocessingStateKeys.LOG2_TRANSFORMED]:
+            mat_transpose = mat_transpose.transform(lambda x: np.log2(x))
```

Comment:
Should we put up a streamlit message that we log-transformed it under the hood? Otherwise, it isn't transparent and won't be reproducible in a standalone analysis without looking at the source code

Reply:
I would make it transparent by the axis labelling and as static text on the interface (calling back to streamlit from here would be a hassle). In the long run we can make it optional, in case someone wants to run t-tests on already normally distribtued data.